Felienne/hedy
# [UI IMPROVEMENT] Implements admin email substring filtering

**Description**
In this PR we implement the request email substring filtering on the admin page as well as the manual option to show all users. These are actually two issues, but due to being dependent (a lot) on each other it is best to solve them in one go. We add an additional option to the filtering `select` named "email". Which can be used for substring filtering on e-mail. Next we add a button called `view all users` which will show all users in the db instead of being limited to 500.

**Fix for**
This PR fixes #1744 and fixed #1745.

**How to test**
Make sure you are set as the admin user in `os.getenv('ADMIN_USER')` and navigate to the `/admin` page. Notice a new option in the filtering dropdown menu called "email". Select this and enter any desired substring. Also notice the new button enabling showing all users: 

**Notice: Viewing all users might make the page slow as there are already a lot of users.**

![Schermafbeelding 2022-01-24 om 09 26 43](https://user-images.githubusercontent.com/38155424/150747452-1228a392-f1ea-4ec7-9f8a-08d22f2eb8f1.png)
